modules/ROOT/pages/tuning-repeatable-nonrepeatable.adoc

```diff
@@ -0,0 +1,84 @@
+= Repeatable vs. Non-Repeatable Streaming
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Understanding streaming strategies and properly establishing which one applies best depending on your use case, is key to get the most of your flows in terms of performance.
+
+To determine the streaming strategy that best suits your use case, first identify if you need:
+
+* A non-repeatable stream (read payload only once) or
+* A repeatable stream (read payload more than once)
+
+== Non-Repeatable Streaming
+
+If you can handle your payload as a non-repeatable stream, disable this feature by enabling the following attribute in the connector:
```

Comment:
This is unclear: what does it mean to be able to "handle your payload as a nonrepeatable stream"? Do we mean if your flow is brief enough? only used once? What makes it "handleable," and what do we mean by "handle"?

More importantly, what feature is being disabled? If "repeatable" is the default, then we should state that and clarify that you can CHANGE it to "nonrepeatable," without referring to vague "disable this feature" and "enable this attribute". It's much simpler and clearer to say something like 
```suggestion
If your payload can be processed quickly and one time, replace the connector attribute `repeatable` with `non-repeatable`:
```

Reply:
Can we provide an example of such a payload?